features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle_rac.sql

```diff
@@ -1674,3 +1718,11 @@ CREATE INDEX IDX_FEDERATED_AUTH_SESSION_ID ON IDN_FED_AUTH_SESSION_MAPPING (SESS
 -- IDN_REMOTE_FETCH_REVISIONS --
 CREATE INDEX IDX_REMOTE_FETCH_REVISION_CONFIG_ID ON IDN_REMOTE_FETCH_REVISIONS (CONFIG_ID)
 /
+
+-- IDN_CORS_ASSOCIATION --
```

Comment:
These indexes will not work Since the identifier length is too high

Reply:
Fixed